charts/alfresco-search-service/templates/secret-repository.yaml

```diff
@@ -1,9 +1,8 @@
-{{- if (not .Values.repository.existingSecret.name) }}
-{{- if and (eq "secret" .Values.repository.securecomms) (not .Values.repository.existingConfigMap.name) }}
+{{- if and (not .Values.repository.existingSecret.name) (eq "secret" .Values.repository.securecomms) }}
```

Comment:
double check, I removed `not .Values.repository.existingConfigMap.name`

Reply:
I think it was there because if the repo is configured through a cm then the securecomms is not accessible to check whether is set to `secret`.  So if the cm is configured with `securecomms=none` the secret would still be created by the chart (as the default value set i s `seret`)

Reply:
at this point I am not sure it's a good idea to have `securecomms` inside the configmap, what's the reason for it? it's not something dynamic like hostname/port